src/templates/PoolManager/ActivityCard/TransactionInfo/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { actionsType } from '..'
+
+import UserInfo from '../UserInfo'
+import TokenInfo from '../TokenInfo'
+
+import * as S from './styles'
+
+type ITransactionDetailsProps = {
```

Comment:
I guess it should not have the suffix `Props`, it should be only `TransactionDetails`, in React we tend to use Props only for a component props